src/sentry/conf/server.py

```diff
@@ -679,22 +679,13 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue("get_suspect_resolutions", routing_key="get_suspect_resolutions"),
     Queue("get_suspect_resolutions_releases", routing_key="get_suspect_resolutions_releases"),
     Queue("replays.delete_replay", routing_key="replays.delete_replay"),
+    Queue("counters-0", routing_key="counters-0"),
+    Queue("triggers-0", routing_key="triggers-0"),
```

Comment:
not sure if `exchange` still should be passed in here

Reply:
I think we're fine without it, these are just normal queues now